script.js

```diff
@@ -284,4 +289,13 @@ allBtns.forEach((btn) => {
     popupMain.style.display = 'flex';
     document.querySelector('.popupMainDiv').style.display = 'flex';
   });
+});
+
+form.addEventListener('submit', (event) => {
+  // event.preventDefault();
```

Comment:
- [ ] Everything works well. Remember, you were advised to follow clean code practices. Please kindly remove the commented code.